### DISCUSSION TITLE
Comprehensive list of native front-matter meta tags?

### DISCUSSION THREAD

**Discussion by Spunkie** [nuxt/content#2718] [Q&A]
The docs(https://content.nuxt.com/usage/markdown#front-matter) say that `title`, `description`, `draft`, `navigation`, and `head` are the only native parameters for front-matter.

But if I put an `image` tag into my front-matter content files like this
```
---
title: 'Tester'
description: 'It's a test.'
image: '/images/tester.png'
---
```
nuxt does render it properly into a `og:image` meta tag:
```html
<head>
    <title>Tester</title>
    <meta property="og:title" content="Tester" />
    <meta name="description" content="It's a test." />
    <meta property="og:description" content="It's a test." />
    <meta property="og:image" content="/images/tester.png" />
</head>
```

So where can I find an actual comprehensive list of native front-matter parameters?

**Comment by nobkd** [comment]
Here are more, but I don't know if the list is exhaustive: https://content.nuxt.com/composables/use-content-head/#parameters

I'd think the relevant code is here, but that was just a quick search: https://github.com/nuxt/content/blob/main/src/runtime/composables/head.ts

Good luck.
